cmd/prysmctl/validator/withdraw.go

```diff
@@ -9,15 +9,25 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/ethereum/go-ethereum/common"
 	"github.com/logrusorgru/aurora"
 	"github.com/pkg/errors"
 	"github.com/prysmaticlabs/prysm/v3/api/client/beacon"
 	"github.com/prysmaticlabs/prysm/v3/beacon-chain/rpc/apimiddleware"
+	fieldparams "github.com/prysmaticlabs/prysm/v3/config/fieldparams"
 	log "github.com/sirupsen/logrus"
 	"github.com/urfave/cli/v2"
 	"go.opencensus.io/trace"
 )
 
+type cliStakingWithdrawalMessage struct {
```

Comment:
I think using `apimiddleware.SignedBLSToExecutionChangeJson` directly should still work if we don't need these additional fields. They will simply be ignored.

Reply:
I guess this brings the question do we want to use it, maybe i should be displaying it for error purposes?